### PR TITLE
Python3 configuration

### PR DESCRIPTION
**Pull request by pferate**
Setting up test environment for 3.3 and 3.4.  Expect lots of red for now.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -16,3 +16,9 @@ commands = nosetests --ignore-files=test_oauth2client_appengine\.py
 
 [testenv:py27]
 commands = nosetests
+
+[testenv:py33]
+commands = nosetests
+
+[testenv:py34]
+commands = nosetests
```
Comment (nathanielmanistaatgoogle) [inline tox.ini]: Please retain the newline at end-of-file.

**Comment by nathanielmanistaatgoogle** [comment]
Diff looks good but commit chain looks really weird - let's get that cleaned up.

I'm disinclined to redden the master build of the project; would it be okay to keep this change on ice and keep updating it as the prerequisite changes to the rest of the codebase land?

**Comment by pferate** [comment]
Initially, I pulled through github's web interface to update my master branch, which put in that extra commit.

I rolled back and redid the commits through CLI, so that it looks cleaner now.

I'm okay with holding off on merging this for now, I just did it so that I could see the errors from tox and travis; the only red would be from the python 3 tests until we were finished.

**Comment by pferate** [comment]
@nathanielmanistaatgoogle: I created my other branch (PR #64) off of this one, so it also includes these two commits.

I suppose it's up to you how you would like to merge these two PRs.

**Comment by nathanielmanistaatgoogle** [comment]
(This pull request also smells obviated in light of #64 and like it should be closed.)

**Comment by pferate** [comment]
Not quite.  This PR is for adding the Python 3.3 and 3.4 tests to tox (and in turn travis).

I will rebase this branch and let you know when it's ready to merge.

Should I move the IDE additions in .gitignore to another PR, or would you just want to keep it out of the master branch all together?

**Comment by nathanielmanistaatgoogle** [comment]
My first reaction is to say that the IDE additions to .gitignore should be kept out of the master branch altogether because they strike me as one person's personal development circumstances and thus not appropriate for a shared codebase. Are there other developers who would want those additions? Are they so insignificant that I shouldn't even bother caring?

**Comment by pferate** [comment]
Understandable.  This PR is now just the one commit that adds the test environments to tox.

**Comment by nathanielmanistaatgoogle** [comment]
Any idea what's up with Travis taking forever to test this?

**Comment by pgrosu** [comment]
Looks like it just started now based on this:

https://travis-ci.org/google/google-api-python-client/builds/53837903

**Pull request merged**